Edits on manuscript (#42)
popsim-consortium/analysis2_manuscript: analysis2_manuscript.tex

\newcommand{\rngcomment}[1]{\textcolor{red}{RNG: #1}}

\newcommand{\kelcomment}[1]{\textcolor{blue}{KEL: #1}}

\title{Adding selection to \stdpopsim: oh how the tables have turned}

>     \kelcomment{Normally, I like quirky, witty titles, but I'm not sure I get this one...what is "how the tables have turned" in reference to? } 

Reviewer: yeah this was just a place holder-- we need a real title!

    For this simulation we have used
    a genetic map from the HapMap Project \citep{international2007second} along with a
    DFE for deleterious alleles (estimated by \cite{huber2017determining})

>     that acts on exonic \kelcomment{do you mean exonic, or nonsynonymous?} positions defined from the HAVANA group release 104 for the human genome.

Reviewer: so selected mutations are falling on exons- there is no sense of synonymous or nonsyn sites within the simulations

    % DRS: Maybe we should say something about how easy it was to do these simulations, and maybe even show
    % some example code somewhere to prove our point? Could help emphasize the utility of this resource.
    These simulations were then used to estimate $N_e(t)$ using four methods: 1) \msmc \citep{Schiffels2020}, 
    2) \stairway \citep{liu2020stairway}, 3) \gone \citep{santiago2020recent}, and 4) \smcpp \citep{terhorst2017robust}.
    In this and what follows we performed 3 replicate whole genome simulations, so uncertainty in estimates might
    be roughly assessed.

>     However, as a thorough benchmarking is not the main purpose of this manuscript, we saved computational resources by not conducting more simualtion replicates.

Reviewer: ```suggestion
    However, as a thorough benchmarking is not the main purpose of this manuscript, we saved computational resources by not conducting more simulation replicates.
```

    the performance of different methods for inferring the DFE. We performed simulations using the 
    human and vaquita porpoise models described above, with complex demographic histories and models of selection acting on exons.
    

>     A challenge when comparing quantitative DFE estimates among tools and with simulations is differing conventions of factors of two in terms of how the selection coefficient ($s$) is delimited.

Reviewer: ```suggestion
    A challenge when comparing quantitative DFE estimates among tools and with simulations is differing conventions of factors of two in terms of how the selection coefficient ($s$) is defined.
```

    With respect to the strength of selection, as in constant size populations,
    all methods are performing reasonably well, although \polydfe seems to
    lead in terms of accuracy.

> \kelcomment{I'm a little surprised by the non-great performance of Dadi-cli in here....I know that folks have spent a long time working on this analysis and debugging it, so I don't want be negative or delay things. That said, I think it would be good to plot the empirical (in the case simulated) SFS along with the SFSs from the MLE models. I would also do this for the demogrpahic models as well as the DFE models}

Reviewer: yeah we all were...

    
    In Figures \ref{fig:vaquita-dfe.constant} and \ref{fig:vaquita-dfe} we show a comparison of estimates
    from the same three DFE-inference packages on simulations of the

>     vaquita porpoise genome. Importantly, these simulations use the vaquita demogrpahic model shown in Fig. X and a DFE that includes recessive deleterious mutations, as suggested in Robinsion et al. (2022). All methods

Reviewer: ```suggestion
    vaquita porpoise genome. Importantly, these simulations use the vaquita demogrpahic model shown in Fig. \ref{fig:1pop-vaquita-demography} and a DFE that includes recessive deleterious mutations, as suggested in \cite{robinson2022critically}. All methods
```
